static/app/components/forms/model.tsx

```diff
@@ -200,6 +202,13 @@ class FormModel {
     return this.fields;
   }
 
+  /**
+   * Is form invalid
+   */
+  get isInvalid() {
+    return this.formState === FormState.INVALID;
+  }
```

Comment:
What is the difference between this and the form having errors?

Reply:
When the form contains errors, they are shown within the fields. However, we prefer not to display these errors immediately upon opening the form